index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+  <title>Ahmad Zamir Yousufi Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <img class="logo" alt="logo" src="images/zheaderlogo.png">
+    <nav><img class="victor" alt="minue" src="images/Union.svg"></nav>
+  </header>
+  <main>
+    <section class="top">
+      <h1 class="hfirst">Hey There.<br>
+        Im Zamir</h1>
+      <h2 class="hsecond">I am a Software Developer</h2>
+      <p class="paragraph"> I am a doctor and a software developer i can help you build a product. feature or website
+        look through some of my work and experience! if you like what you see and have a project you need coded, dont
+        hesitate to contact me.</p>
+      <div class="social">
+        <a href="https://github.com/ahzamir" target="blank"><img class="icon" alt="github" src="images/git.svg"></a>
+        <a href="https://www.linkedin.com/in/ahmad-zamir-yousufi-70603317b/" target="blank"><img class="icon" alt="linkedin" src="images/linkedin.svg"></a>
+        <a href="#" target="blank"><img class="icon" alt="angel" src="images/angel.svg"></a>
+        <a href="https://twitter.com/YousufiZamir" target="blank"><img class="icon" alt="twitter" src="images/twitter.svg"></a>
+        <a href="#" target="blank"><img class="icon" alt="medium" src="images/medium.svg"></a>
+      </div>
```

Comment:
Kindly use semantic HTML tags like `<ul>` `<li>` for displaying list of elements. Please replace `<div>` `<a>` tags with `<ul>` `<li>` tags.